tests/test_flow/test_persistence_aip.py

```diff
@@ -2,30 +2,50 @@
 
 import bionic as bn
 
-
 # This is detected by pytest and applied to all the tests in this module.
-pytestmark = [pytest.mark.no_parallel]
+pytestmark = pytest.mark.needs_aip
 
 
-@pytest.mark.needs_aip
 def test_aip_jobs(aip_builder):
```

Comment:
Do we have any tests that check that the `aip_task_config` tasks were actually sent to AIP (or to a fake AIP)? If not, could we add a TODO for that?

On a similar note: are two test cases (this one and `test_logging`) enough? Are there any other tests we should come back and add? For example, can/should we have a case where an AIP task and a subprocess task are running in parallel?

Reply:
Expanded test to have functions that can run in parallel as well.